sdks/python/apache_beam/yaml/readme_test.py

```diff
@@ -155,14 +161,27 @@ def __exit__(self, *args):
     self.tempdir.cleanup()
 
 
+def replace_mapping_fields(spec, arg_name, arg_value):
+  if arg_name != 'language' and arg_name != 'fields':
```

Comment:
Can this be phrased positively? I don't know what `arg_name` could be here. Is `language/fields` ever actually passed?